.github/workflows/github-actions.yml

```
@@ -100,6 +100,10 @@ jobs:
             cd /home/ubuntu/bongbaek-server
 
             export DOCKER_USERNAME=${{ secrets.DOCKER_USERNAME }}
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)